Sources/AST/Component/Identifier.swift

```diff
@@ -11,6 +11,7 @@ import Lexer
 public struct Identifier: Hashable, ASTNode {
   public var identifierToken: Token
   public var enclosingType: String?
+  public var isFunctionCallArgLabel: Bool
```

Comment:
Should this really be state as part of the `Identifier` `ASTNode`? 
We use identifiers in a lot of places and are able to distinguish them later just fine. Can we modify the pass instead to make this work?